src/Tizen.NUI.Components/Controls/RecyclerView/GridRecycleLayoutManager.cs

```diff
@@ -0,0 +1,261 @@
+/* Copyright (c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using Tizen.NUI.BaseComponents;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// [Draft] This class implements a grid box layout.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class GridRecycleLayoutManager : RecycleLayoutManager
+    {
+        private float mLayoutOriginPosition = 0;
+
+        private int row = 1;
+
+        /// <summary>
+        /// [draft ]Get/Set the number of rows in the grid
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int Row
+        {
+            get
+            {
+                return row;
+            }
+            set
+            {
+                row = value;
+
+                if (Container != null)
+                {
+                    Layout(mPrevScrollPosition);
+                }
+            }
+        }
+
+        private int column = 1;
+
+
+        /// <summary>
+        /// [Draft] Get/Set the number of columns in the grid
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int Column
```

Comment:
Generally, `Column` is used as a word to represent specific column index of child.
I recommend `Columns` or `ColumnCount`.